### ISSUE TITLE
Can't set a config

### ISSUE THREAD

**Issue by dpkg-i-foo-deb** [MusiKid/fancy#11]
I moved from Gnome to Openbox, when I was trying to set my config, I get the following error:

```fancy set -c "HP EliteBook 8470p"```

```Error: D-Bus error: Message recipient disconnected from message bus without replying (org.freedesktop.DBus.Error.NoReply)```

I guess I'm missing to install something...

**Comment by saidsay-so** [comment]
Do you have something if you run `sudo systemctl status fancy.service`?

**Comment by dpkg-i-foo-deb** [comment]
```
● fancy.service - Fancy Service Daemon
     Loaded: loaded (/lib/systemd/system/fancy.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Thu 2021-09-23 14:10:00 -05; 2min 33s ago
    Process: 67669 ExecStart=/usr/bin/fancyd (code=exited, status=101)
   Main PID: 67669 (code=exited, status=101)
        CPU: 4ms

Sep 23 14:10:00 EliteBook-8470p systemd[1]: fancy.service: Scheduled restart job, restart counter is at 5.
Sep 23 14:10:00 EliteBook-8470p systemd[1]: Stopped Fancy Service Daemon.
Sep 23 14:10:00 EliteBook-8470p systemd[1]: fancy.service: Start request repeated too quickly.
Sep 23 14:10:00 EliteBook-8470p systemd[1]: fancy.service: Failed with result 'exit-code'.
Sep 23 14:10:00 EliteBook-8470p systemd[1]: Failed to start Fancy Service Daemon.

```

**Comment by saidsay-so** [comment]
It seems that it crashed for some reason. Can you share the last error that you can see with `journalctl -xeu fancy.service`?

**Comment by dpkg-i-foo-deb** [comment]
Solved! I don't know what caused the issue, but I solved it by running `systemctl reset-failed fancy.service` and it started again after a reboot :D.

**Comment by dpkg-i-foo-deb** [comment]
It is happening once again...

```fancy set -c "HP EliteBook 8470p"```


```Error: D-Bus error: Message recipient disconnected from message bus without replying (org.freedesktop.DBus.Error.NoReply) ```

```systemctl status fancy.service ```

```● fancy.service - Fancy Service Daemon
     Loaded: loaded (/usr/lib/systemd/system/fancy.service; enabled; vendor preset: disabled)
     Active: active (running) since Wed 2021-09-29 14:44:33 -05; 1min 28s ago
   Main PID: 2112 (fancyd)
      Tasks: 1 (limit: 19070)
     Memory: 620.0K
        CPU: 9ms
     CGroup: /system.slice/fancy.service
             └─2112 /usr/bin/fancyd

sep 29 14:44:33 EliteBook-8470p systemd[1]: Starting Fancy Service Daemon...
sep 29 14:44:33 EliteBook-8470p systemd[1]: Started Fancy Service Daemon.
```

```journalctl -xeu fancy.service```

```Support: https://forum.manjaro.org/c/support

A stop job for unit fancy.service has finished.
 
The job identifier is 1744 and the job result is done.
sep 29 14:44:33 EliteBook-8470p systemd[1]: Starting Fancy Service Daemon...
Subject: A start job for unit fancy.service has begun execution
Defined-By: systemd
 Support: https://forum.manjaro.org/c/support
 
 A start job for unit fancy.service has begun execution.
 
 The job identifier is 1744.
sep 29 14:44:33 EliteBook-8470p systemd[1]: Started Fancy Service Daemon.
 Subject: A start job for unit fancy.service has finished successfully
 Defined-By: systemd
 Support: https://forum.manjaro.org/c/support
 
 A start job for unit fancy.service has finished successfully.
 
 The job identifier is 1744.
```

**Comment by saidsay-so** [comment]
It seems to be the bug that I fixed at 3cda9bf0e2abc8f97be09dd0fcf93a6d9a43c843.
Don't worry, I will push a release which fix it ASAP!

**Comment by saidsay-so** [comment]
You should be able to change your configuration now with the new release.
